File: Services/enerser.analytics/Models/Report/admin_report_update_request.cs
namespace enerser.analytics.Models.Report
{
    public class AdminReportUpdateRequest
    {
        public int id { get; set; } = new int();
        public string name { get; set; } = string.Empty;
        public string url { get; set; } = string.Empty;
        public int report_type_id { get; set; } = new int();
        public int status_id { get; set; } = new int();
    }
}

File: Application Mobile/lib/Models/AdminReports/admin_report_update_request.dart
class AdminReportUpdateRequest {
  int id;
  String name;
  String url;
  int reportTypeId;
  int statusId;

  AdminReportUpdateRequest({
    this.id = 0,
    this.name = '',
    this.url = '',
    this.reportTypeId = 0,
    this.statusId = 0,
  });


  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'url': url,
      'report_type_id': reportTypeId,
      'status_id': statusId,
    };
  }
}